src/modules/notifications/selectors/notification-state.js

```diff
@@ -145,6 +155,43 @@ export const selectProceedsToClaim = createSelector(
   }
 );
 
+// Get all markets where the user has outstanding returns but needs to wait CLAIM_PROCEEDS_WAIT_TIME
+export const selectProceedsToClaimOnHold = createSelector(
+  selectAllProceedsToClaim,
+  selectCurrentTimestampInSeconds,
+  (markets, currentTimestamp) => {
+    if (markets.length > 0 && currentTimestamp) {
+      return markets
+        .filter(
+          market =>
+            !canClaimProceeds(
```

Comment:
Looks like this won't return markets where the waiting period hasn't passed.

don't you want all markets where user has winning shares `outstandingReturns`

Reply:
I created three selectors to manage this.

First ```selectAllProceedsToClaim ``` gets all finalized markets with outstanding shares.

Second ```selectProceedsToClaim``` gets all markets that are claimable

and finally ```selectProceedsToClaimOnHold``` gets all markest that need to wait until claimable 

Really ```selectProceedsToClaim``` and ```selectProceedsToClaimOnHold``` are just the same but inverted.

Reply:
// Get all markets where the user has outstanding returns but needs to wait 

this comment, It seems you don't want to filter out markets that user has winning shares, but will need to wait.

Reply:
Correct, for this selector, I don't want to filter out markets that an user has winning shares for but has it wait. 

This selector does the oppsoite, it filters out the markets that an user has wining share for and are ready to be claim right now.

I do this by grabbing all the markets with winning shares and filtering on all the markets that **can't** canClaimProceeds yet.

```
        .filter(
          market =>
            !canClaimProceeds(...
```